google/cloud/documentai_toolbox/wrappers/document_revision.py

```diff
@@ -0,0 +1,346 @@
+# -*- coding: utf-8 -*-
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Wrappers for Document AI Document type."""
+
+import dataclasses
+import re
+import copy
+
+from typing import List, TypeVar
+
+from google.cloud import documentai
+from google.cloud import storage
+
+from google.cloud.documentai_toolbox.wrappers.entity import Entity
+
+from google.cloud.documentai_toolbox.wrappers.document import Document
+
+_OP_TYPE = documentai.Document.Provenance.OperationType
+
+DocumentWithRevisions = TypeVar("DocumentWithRevisions")
+
+
+def _get_base_and_revision_bytes(output_bucket: str, output_prefix: str) -> List[bytes]:
```

Comment:
I don't think the return type is correct. You're returning a Tuple with a `string, list[bytes], list[bytes]`

Reply:
fixed.